### PR TITLE
OSDOCS-1865: RN: Tracking network flows

### PR DESCRIPTION
**Pull request by mikemckiernan**
Release notes PR.

Please review [Tracking network flows](https://deploy-preview-32099--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-tracking-network-flows) in the release notes.

The link (and the build) are broken until #32089 is merged.

----
Applies to branch enterprise-4.8 and milestone Future-Release.

### CODE REVIEW

**Comment by mikemckiernan** [comment]
/assign @rbbratta 
/assign @rcarrillocruz 

Please review.

Ricky, if you don't have time to review, please suggest a name. Thanks.

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -103,6 +103,21 @@ The Network Resources Injector that is deployed with the Operator is enhanced to
 
 For more information, see xref:../networking/hardware_networks/about-sriov.adoc#nw-sriov-hugepages_about-sriov[Huge pages resource injection for Downward API].
 
+[id="ocp-4-8-tracking-network-flows"]
+==== Tracking network flows
+
+{product-title} {product-version} adds support for sending the metadata about network flows on the pod network to a network flows collector. The following protocols are supported:
+
+* NetFlow
+
```
Comment (skrthomas) [inline release_notes/ocp-4-8-release-notes.adoc:112]: Is the spacing in the list necessary? When looking at the list in the RN above, it doesn't look like there's a space.

**Review by skrthomas — COMMENTED** [review]
One question about spaces. LGTM otherwise :) Nice work!

**Pull request merged**